doc/commands.md

```diff
@@ -188,6 +190,18 @@ And they can be non-linear:
 rule minus x x ↪ zero
 ```
 
+Note that rewriting rules can also be defined simultaneously, using the `with`
+keyword instead of the `rule` keyword for all but the first rule.
+
+```
+rule add zero      $n ↪ $n
+with add (succ $n) $m ↪ succ (add $n $m)
+```
+
+Adding sets of rules allows to maintain confluence.
+
+Examples of patterns are available in the file `patterns.lp` of the test suite.
```

Comment:
Add a link to https://github.com/Deducteam/lambdapi/blob/master/tests/OK/patterns.lp ?

Reply:
OK, it worked.